src-web/actions/topology.js

```diff
@@ -472,7 +472,7 @@ const fetchArgoApplications = (
           propValue = 'HEAD'
         }
       }
-      if (property !== 'helm') {
+      if (property !== 'helm' && property !== 'directory') {
```

Comment:
Search is only indexing the properties we set it up to index:
https://github.com/open-cluster-management/search-collector/blob/main/pkg/transforms/argoapplication.go#L72

I would suggest that we only include these properties. This is what I did for grouping by source for the table:
https://github.com/open-cluster-management/application-ui/blob/adffab874efab1251ec089f572f2d268e16bae77/src-web/definitions/hcm-applications.js#L48

Reply:
Thanks @KevinFCormier. The changes are made.